app_grim_reefer/prj.conf

```diff
@@ -1,33 +1,67 @@
-# Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2024 Launch Initiative
 # SPDX-License-Identifier: Apache-2.0
 #
 # This file contains selected Kconfig options for the application.
 
+CONFIG_EVENTS=y
+CONFIG_RING_BUFFER=y
+CONFIG_BASE64=y
+
 # Random
 CONFIG_TEST_RANDOM_GENERATOR=y
 
 # GPIO for LEDS and enable lines
 CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_I2C=y
+
+# Sensor Enables
+CONFIG_ADC=y
+CONFIG_MCP356X=y
+CONFIG_LSM6DSL=y
+CONFIG_BME280=y
+CONFIG_INA260=y
+
+
+# Sensor Configuration 
+
+
+# BME280 (closely follows 'gaming mode' in data sheet)
+CONFIG_BME280_STANDBY_05MS=y
+CONFIG_BME280_TEMP_OVER_1X=y
+CONFIG_BME280_HUMIDITY_OVER_1X=y
+
 
+# LSM6DSL - 1660Hz for both accel and gyro
+CONFIG_LSM6DSL_ACCEL_ODR=8
+CONFIG_LSM6DSL_GYRO_ODR=8
+
+# 16g scale. 2000 dps 
+CONFIG_LSM6DSL_ACCEL_FS=16
+CONFIG_LSM6DSL_GYRO_FS=2000
+
+
+# Serial for data dumping
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
 
 # Filesystem
+CONFIG_SPI_NOR_SFDP_RUNTIME=y
 CONFIG_FLASH=y
 CONFIG_FLASH_MAP=y
 CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_DISK_ACCESS=y
 CONFIG_FILE_SYSTEM=y
 CONFIG_FILE_SYSTEM_LITTLEFS=y
-CONFIG_FS_LITTLEFS_FMP_DEV=y
-# CONFIG_FS_LITTLEFS_BLK_DEV=y
 
-# needed for file system shell
-CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_FS_LITTLEFS_FMP_DEV=y
+CONFIG_FS_LITTLEFS_BLK_DEV=n
 
-CONFIG_SHELL=y
-# CONFIG_SENSOR_SHELL=y
-CONFIG_GPIO_SHELL=y
-CONFIG_FLASH_SHELL=y
-# CONFIG_FILE_SYSTEM_SHELL=y
-CONFIG_TIMESLICE_SIZE=50
+CONFIG_FS_LITTLEFS_NUM_FILES=6
+CONFIG_FS_LITTLEFS_CACHE_SIZE=2048
+CONFIG_FS_LITTLEFS_LOOKAHEAD_SIZE=128
 
+# needed for file system shell
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=4096
```

Comment:
Recommend testing 2048. See if you can halve the stack size of main. Was able to get away with this for initing littlefs

Reply:
I left this kinda big since main is stripped back for the PR. 4096 is a value needed for the test flight.